Src/BootCamp.Chapter1/ArrayOperations.cs

```diff
@@ -75,8 +134,22 @@ public static int[] InsertFirst(int[] array, int number)
         /// <returns>A new array with element added in the end of array. If an array is empty or null, returns new array with number in it.</returns>
         public static int[] InsertLast(int[] array, int number)
         {
-            // ToDo: implement.
-            return array;
+            // Returns new number array if array is empty of null.
+            if (array == null || array.Length == 0) return new int[1] { number };
+
+            // Create new array with an extra slot
+            int[] new_array = new int[array.Length + 1];
+
+            // If at last position, add the number, else add value from old array
+            for (int i = 0; i < new_array.Length; i++)
+            {
+                if (i == array.Length)
```

Comment:
Is there a way to write this that doesn't require a branch comparison of `i` to `array.Length` each iteration of the loop?